specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -472,71 +417,75 @@ namespace Microsoft.Windows.PushNotifications
     };
 
     // The PushNotificationChannel Progress result
+    [feature(Feature_PushNotifications)]
     struct PushNotificationCreateChannelStatus
     {
-        // The last extended error we failed Channel requests on that caused the inprogress retry status. S_OK if this is just progress status.
-        HRESULT extendedError;
-
         // Either InProgress or InProgressRetry status
         PushNotificationChannelStatus status;
 
+        // The last extended error we failed Channel requests on that caused the inprogress retry status. E_PENDING if this is just progress status.
```

Comment:
Is this really the "last extended error"? If so maybe rename to lastExtendedError so it's clear that other errors are overwritten. 

Can we also re-word to  something a little more clear such as - "// The last extended error seen when creating a channel request. This error causes InProgressRetry status. If there is no retry and just InProgress status this defaults to E_PENDING"

Reply:
Fixing the comment and tracking bug to rename the param. https://github.com/microsoft/WindowsAppSDK/issues/2229